docbot/main.go

```diff
@@ -309,6 +309,11 @@ func (a *Action) onPullRequestOpenedOrEdited() error {
 		}
 	}
 
+	if _, exist := currentLabelsSet[a.config.GetLabelMissing()]; exist && checkedCount == 0 {
```

Comment:
Please move this section to line 326.

Reply:
I can move this section inside `if a.config.GetEnableLabelMissing() && checkedCount == 0`, but cannot after `AddLabelsToIssue`. Otherwise, the docbot will never create comment because we check label missing just after we add it, and it will be always true.

Reply:
Updated at 1cab1c23a53446c107d594f177e945e5ffa09733